modules/pdf2md.py

```
@@ -79,65 +79,70 @@ def save_images(images: dict, image_dir: Path) -> None:
+
+        # Build page_range config. marker-pdf 1.x requires an explicit list of
+        # page indices; there is no "start to end" shorthand. If start_page is
+        # given without max_pages we cannot construct the range without knowing
+        # the document's page count, so we warn and process all pages instead.
+        config = {}
+        if max_pages is not None:
+            s = start_page or 0
+            config["page_range"] = list(range(s, s + max_pages))
+        elif start_page is not None:
+            print(
+                f"Warning: --start-page requires --max-pages in marker-pdf 1.x "
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)